guides/v2.3/extension-dev-guide/declarative-schema/index.md

```diff
@@ -28,8 +28,8 @@ Once a module is converted to the declarative schema approach, it cannot be reve
 
 * *Data patch* - A class that contains data modification instructions. It can have dependencies on other data or schema patches.
 
-* *Revertable data patch* - A patch that can be reverted as a module or patch is uninstalled or deleted. Revertable operations are Data Query Language (DQL) and Data Manipulation Language (DML) operations: INSERT, UPDATE.
-
+* *Revertable data patch* - A `data patch` that contains a `revert()` method (see [Reverting data patches]({{ page.baseurl }}/extension-dev-guide/declarative-schema/data-patches.html/#revertingDataPatches)) with provided operations that revert revertable changes caused by the module. Revertable operations are usually related to changes in the database.
```

Comment:
Does the link actually work with the `/` before the anchor?

```suggestion
* *Revertable data patch* - A `data patch` that contains a `revert()` method (see [Reverting data patches]({{ page.baseurl }}/extension-dev-guide/declarative-schema/data-patches.html#revertingDataPatches)) with provided operations that revert revertable changes caused by the module. Revertable operations are usually related to changes in the database.
```

Reply:
Tests passed, so I think the answer is yes.

Reply:
Just confirmed that this works without the slash. It's already that way on production: https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/data-patches.html#reverting-data-patches